### ISSUE TITLE
Usability: NVDA is not reciting the name for the added snippet into the program.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17236]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: Node.js and git repository need to be installed.
1.Open Visual studio code editor.
2.Open any file and press enter key to give some space.
3.Using "Ctrl+ Space" key open the suggestion and Type "For".
4.Using the Down arrow key select the "For loop " option.
5.Using keyboard select the value for the for loop section.

**Expected result:**
NVDA should recite the name for the added snippet into the code.
Ex:-If the user is adding "For loop" snippet into the program then NVDA should recite like "For Loop" suggestion and after entering then it should announce "For loop" added .

Actual Result:
NVDA is not reciting the name for the added snippet into the program.
Ex: If user enters "For" snippet to the program then NVDA is announcing like "for suggestion for(var index selected index)"

User Impact:
Screen Reader user doesn't get to know which code has been added as the Name of the code is not announced

Attachments:
![nvda_speechviewerinformation_vscodeeditor_snippet](https://cloud.githubusercontent.com/assets/22956130/21212908/d6e63d58-c2b6-11e6-97ed-2bf4bc630681.jpg)
